samples/cpp/speech_sample/main.cpp

```diff
@@ -221,14 +224,18 @@ int main(int argc, char* argv[]) {
             executableNet = core.compile_model(model, deviceStr, genericPluginConfig);
         } else {
             slog::info << "Importing model to the device" << slog::endl;
-            std::istringstream streamrq(FLAGS_rg);
+            std::ifstream streamrq(FLAGS_rg, std::ios_base::binary | std::ios_base::in);
```

Comment:
What if --rg is absent?

Reply:
We have checking in function parse_and_check_command_line: "if (FLAGS_m.empty() && FLAGS_rg.empty())"